src/main/java/com/danielagapov/spawn/Services/User/UserService.java

```diff
@@ -46,13 +44,9 @@ public UserDTO getUserById(UUID id) {
     }
 
     public List<UserDTO> getUsersByTagId(UUID tagId) {
```

Comment:
Just a note for this file, based on the logic I wanted you to move out of the mapper files, these service classes is where the logic shouuld be put, when you're calling the `.toDTO()` or `.toDTOList()` methods